content/pt-br/pod.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Pod
+status: Completed
+category: conceito
+tags: ["infraestrutura", "fundamento", ""]
+---
+
+Em um ambiente [Kubernetes](/kubernetes/), um pod atua como a unidade de implantação mais básica.
```

Comment:
When the page exists in Portuguese, you must change the link to point to the pt-br version.
Example: `[Kubernetes](/kubernetes/)`-> `[Kubernetes](/pt-br/kubernetes/)`

You can do this to all the links.

Reply:
Done ✅